Updates for new tnu table features

xbrianh opened the pull request:
This uses [terra-notebook-utils](https://github.com/DataBiosphere/terra-notebook-utils) Terra data table features to simplify [xvcfmerge](https://github.com/DataBiosphere/xvcfmerge) input table creation.

Note that this notebook supersedes [VCF Merge and Subsample Tutorial](https://github.com/DataBiosphere/featured-notebooks/blob/master/notebooks/vcf_merge_subsample_tutorial/main.py), which should be deleted from the BioData Catalyst collection when this PR is merged, and when the [xvcfsubsample](https://github.com/DataBiosphere/xvcfsubsample) example notebook is complete.

bethsheets reviewed (changes requested):
This looks good! Some requests:
1) Can you make it clear that you can combine gs:// and drs:// inputs in the workflow? Either as a statement or an option C?
2) Tangental, but I made this error: for users that attempt to decide to skip the data table creation and do direct input, can you specify either here or in the readme of your xvcfmerge workflow the accepted input file types and outfile format requirements? 
3) I would remove the ~20 min per VCF timing bit, because my test trials with large data files were more like 24 hours. I added benchmarking data in the readme of this collection: https://dockstore.org/organizations/bdcatalyst/collections/Utilities

xbrianh commented:
@bethsheets 
1. Option C added for mixed input of DRS URI and Google bucket URLs
2. Added file type requirements to notebook description.
3. Replaced misleading runtime information with link to Dockstore BDCat utilities

bethsheets commented on notebooks/xvcfmerge_array_input/main.py:
```diff
@@ -26,113 +29,152 @@
-    Workflow execution time is typically ~20 minutes per VCF.
+    Runtime and cost can vary greatly depending on the size and number of intput files. Sample runtime and cost
+    information is available for previous merge workflows in
+    [NHLBI BioData Catalyst / Utilities](https://dockstore.org/organizations/bdcatalyst/collections/Utilities) for
```
is the "for" intended to be at the end of this sentence?

xbrianh replied:
I like to sprinkle in random prepositions to keep people on their toes.

will fix.

bethsheets reviewed (approved):
Looks good! Thanks, Brian!
